add sooperlooper

magnetophon opened the pull request:
had to change the license from gpl to gpl1 due to travis complaining.
tried to squash my commits but failed, sorry!

magnetophon committed:
add sooperlooper

cillianderoiste commented:
Isn't it GPL2? https://github.com/essej/sooperlooper/blob/master/COPYING (I believe GPL1 is extremely rare).

cillianderoiste commented on pkgs/top-level/all-packages.nix:
```diff
@@ -12909,6 +12909,8 @@ let
       pysqlite = pkgs.pythonPackages.sqlite3;
   };
 
+  sooperlooper = callPackage ../applications/audio/sooperlooper { };
```
Entries in all-packages.nix should be in alphabetical order and in the appropriate group. You can see the groups by searching for "###" perhaps Applications would be appropriate?

magnetophon commented:
tried to squash my commits again, and failed. again...  :(
this is turning into a bit of a mess, sorry.

cillianderoiste commented on pkgs/applications/audio/sooperlooper/default.nix:
```diff
@@ -0,0 +1,35 @@
+{ stdenv, fetchurl, liblo, libxml2, jack2, libsndfile, wxGTK, libsigcxx,
+libsamplerate, rubberband, pkgconfig, ncurses
```
Please put the "," at the start of the wrapped line.

cillianderoiste commented on pkgs/applications/audio/sooperlooper/default.nix:
```diff
@@ -0,0 +1,35 @@
+  buildInputs = [
+   liblo libxml2 jack2 libsndfile wxGTK libsigcxx
+   libsamplerate rubberband pkgconfig ncurses
+
```
... and remove the extra line break, thanks!

magnetophon committed:
Merge branch 'master' of https://github.com/NixOS/nixpkgs

magnetophon committed:
# This is a combination of 2 commits.
# The first commit's message is:

add sooperlooper

# The 2nd commit message will be skipped:

#	add sooperlooper
